### PR TITLE
Update area owners

### PR DESCRIPTION
**Pull request by steveisok**
Removes @lambdageek

### CODE REVIEW

**Review by jeffschwMSFT — APPROVED** [review]
lgtm

**Review by vitek-karas — APPROVED** [review]

**Review by jeffhandley — APPROVED** [review]

**Review by jeffhandley — COMMENTED** [review]
After marking as approved, I realized 2 things:

1. The CODEOWNERS file names `@mrvoorhe` alongside `@marek-safar` on a couple ILLink codeowners entries; is that still right?
2. The `.github/policies/resourceManagement.yml` file (that controls the repo automation for mentions by area ownership) probably needs to be updated as well.

**Comment by srxqds** [comment]
I found that the number of monovm maintainers is decreasing. I am worried about the subsequent development and support.

**Pull request merged**
